specification/dataprotection/resource-manager/readme.md

```diff
@@ -142,6 +151,7 @@ input-file:
   - Microsoft.DataProtection/stable/2021-01-01/dataprotection.json 
   - Microsoft.DataProtection/preview/2021-02-01-preview/dataprotection.json
   - Microsoft.DataProtection/preview/2021-06-01-preview/dataprotection.json 
+  - Microsoft.DataProtection/preview/2021-10-01-preview/dataprotection.json 
```

Comment:
Do you plan to regenerate the sdk for 2021-07?

Reply:
sorry; didn't get the question: version 2021-07 is not updated

Reply:
Not sure it is the right pattern to put '2021-10-01-preview' into 2021-07 package.
Usually, if we want to support multi api-version, we leverage the approach of multi tag instead of putting the same file with different version under one tag.

Reply:
But in the SDK package of 'package-2021-07',  you added another json file

Reply:
removed the line